modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -25,8 +26,48 @@
  * @since 3.23.0
  */
 class Link_In_Bio extends Widget_Base {
+
+	use Conversion_Center_Controls_Trait;
+
 	const TAB_ADVANCED = 'advanced-tab-links-in-bio';
 
+	public static function get_configuration() {
```

Comment:
Can you please move all the common functionality into an abstract class?
`abstract class Widget_Link_In_Bio_Base extends Widget_Base {}`, similar to what we did with our nested widgets that extend `Widget_Nested_Base`.